source/api/commands/writefile.md

```diff
@@ -130,6 +131,16 @@ cy.writeFile('path/to/ascii.txt', 'Hello World', 'ascii'))
 Hello World
 ```
 
+## Flags
+
+***Append contents to the end of a file***
+
+```javascript
+cy.writeFile('path/to/message.txt', 'Hello World', {flag: "a+"}))
+```
```

Comment:
`Minor` `Style`

Code snippets should prefer single quotes per Jennifer's [WIP style guide](https://github.com/cypress-io/cypress-documentation/issues/853).

```js
cy.writeFile('path/to/message.txt', 'Hello World', { flag: 'a+' })
```

Reply:
`Typo`

There's an extra closing parentheses in that expression that can be culled:

```js
cy.writeFile('path/to/message.txt', 'Hello World', { flag: 'a+' })
```

Reply:
Thanks for the catches! 🎉